src/theory/bv/theory_bv_rewrite_rules_simplification.h

```diff
@@ -890,22 +904,38 @@ Node RewriteRule<NegIdemp>::apply(TNode node) {
 
 template<> inline
 bool RewriteRule<UdivPow2>::applies(TNode node) {
```

Comment:
clang-format whole function including header (since all of it's content changed anyway)